SWLOR.Game.Server/Service/AchievementService/AchievementType.cs

```diff
@@ -299,6 +299,33 @@ public enum AchievementType
 
         [Achievement("The Legendary Rod", "Completed the quest 'The Legendary Rod'.", true)]
         TheLegendaryRod = 110,
+
+        [Achievement("Explore Dantooine Orbit", "Explore Dantooine's Orbit'.", true)]
+        ExploreDantooineOrbit = 111,
+
+        [Achievement("Explore Dantooine Field", "Explore Dantooine's Fields.", true)]
+        ExploreDantooineField = 112,
+
+        [Achievement("Explore Dantooine Colony", "Explore Dantooine's Colony.", true)]
+        ExploreDantooineColony = 113,
+
+        [Achievement("Explore Dantooine Lake", "Explore Dantooine's Lake.", true)]
+        ExploreDantooineLake = 114,
+        
+        [Achievement("Explore Dantooine Kinrath Cave", "Explore Dantooine's Kinrath Tunnels.", true)]
+        ExploreDantooineKinrathTunnel = 115,
+
+        [Achievement("Explore Dantooine Wild Plains", "Explore Dantooine's Wild Plains.", true)]
+        ExploreDantooineWildPlains = 116,
+
+        [Achievement("Explore Dantooine South Plains", "Explore Dantooine's Tribe.", true)]
+        ExploreDantooineTribe = 117,
+
+        [Achievement("Explore Dantooine Abandon Warehouse", "Explore Dantooine's Abandon Base.", true)]
```

Comment:
Abandon -> Abandoned. Also, consistency here too - is it a Warehouse or a Base? Please update to reflect appropriately.